src/test/javascript/spec/component/exam/participate/exam-start-information/exam-start-information.component.spec.ts

```diff
@@ -0,0 +1,189 @@
+import { ExamStartInformationComponent } from 'app/exam/participate/exam-start-information/exam-start-information.component';
+import { InformationBoxComponent } from 'app/shared/information-box/information-box.component';
+import { StudentExamWorkingTimeComponent } from 'app/exam/shared/student-exam-working-time/student-exam-working-time.component';
+import { ArtemisDatePipe } from 'app/shared/pipes/artemis-date.pipe';
+import { ArtemisDurationFromSecondsPipe } from 'app/shared/pipes/artemis-duration-from-seconds.pipe';
+import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
+import dayjs from 'dayjs/esm';
+import { MockComponent, MockDirective, MockPipe } from 'ng-mocks';
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+import { ArtemisExamSharedModule } from 'app/exam/shared/exam-shared.module';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
+
+let fixture: ComponentFixture<ExamStartInformationComponent>;
+let component: ExamStartInformationComponent;
+
+const user = { id: 1, name: 'Test User' } as User;
+
+const startDate = dayjs('2022-02-06 02:00:00');
+const endDate = dayjs(startDate).add(1, 'hours');
+
+let exam = {
+    id: 1,
+    title: 'Test Exam',
+    startDate,
+    endDate,
+    testExam: false,
+} as Exam;
+
+let studentExam = { id: 1, exam, user, workingTime: 60, submitted: true } as StudentExam;
+
+describe('ExamStartInformationComponent', () => {
+    beforeEach(() => {
+        exam = { id: 1, title: 'ExamForTesting', examMaxPoints: 10, startDate, endDate, testExam: false } as Exam;
+        studentExam = { id: 1, exam, user, workingTime: 60, submitted: true } as StudentExam;
+
+        return TestBed.configureTestingModule({
+            imports: [RouterTestingModule.withRoutes([]), ArtemisSharedModule, ArtemisSharedComponentModule, ArtemisExamSharedModule],
+            declarations: [
+                ExamStartInformationComponent,
+                MockComponent(StudentExamWorkingTimeComponent),
+                MockComponent(InformationBoxComponent),
+                MockDirective(TranslateDirective),
+                MockPipe(ArtemisTranslatePipe),
+                MockPipe(ArtemisDatePipe),
+                MockPipe(ArtemisDurationFromSecondsPipe),
+            ],
+        })
+            .compileComponents()
+            .then(() => {
+                fixture = TestBed.createComponent(ExamStartInformationComponent);
+                component = fixture.componentInstance;
+            });
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    it('should initialize', () => {
+        component.exam = exam;
+        component.studentExam = studentExam;
+        fixture.detectChanges();
+        component.ngOnInit();
+        expect(fixture).toBeDefined();
+        expect(component.startDate).toEqual(exam.startDate);
```

Comment:
Maybe the description could be more precise as to what's being tested: should initialize with the correct start date

The fixture is definitely defined - you called `detectChanges` on it -> I'd remove that expect

Do we really need to call ngOnInit after fixture.detectChanges?

```suggestion
    it('should initialize with the correct start date', () => {
        component.exam = exam;
        component.studentExam = studentExam;
        fixture.detectChanges();
        expect(component.startDate).toEqual(exam.startDate);
```

Reply:
You are right, we don't need them.